.github/workflows/veramo-website.yml

```
@@ -0,0 +1,18 @@
+name: Trigger veramo-website build
+on:
+  push:
+    branches:
+      - "main"
+jobs:
+  dispatch:
+    permissions:
+      contents: write
+      actions: write 
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          curl -X POST \
+            -H 'Authorization: Bearer ${{ secrets.GH_TOKEN }}' \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/uport-project/veramo-website/actions/workflows/main.yml/dispatches \
+            -d '{"event_type":"deploy_website"}'
```

I think the repository dispatch event requires another path and they don't mention the Bearer token
https://docs.github.com/en/rest/repos/repos#create-a-repository-dispatch-event

```suggestion
          curl -X POST \
            -H 'Authorization: ${{ secrets.GH_TOKEN }}' \
            -H "Accept: application/vnd.github.v3+json" \
            https://api.github.com/repos/uport-project/veramo-website/dispatches \
            -d '{"event_type":"deploy_website"}'
```